net/softethervpn5/Makefile

```diff
@@ -11,8 +12,8 @@ PKG_LICENSE_FILES:=COPYING
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/SoftEtherVPN/SoftEtherVPN.git
-PKG_SOURCE_DATE:=2019-02-27
-PKG_SOURCE_VERSION:=beef8d68924916d0834c87ceb9ae8eef2c9b1d6c
+PKG_SOURCE_DATE:=2019-04-08
+PKG_SOURCE_VERSION:=b43a1e76da32c7746dfdea85851c9d5434e01978
 PKG_MIRROR_HASH:=8cd27ee1d768c00f816501be925bb971fd62b9cef145e4ac5c731456a86d782c
```

Comment:
This is going to have to change too, according to the build logs

Reply:
forgot to update the hash